org.gnome.Keysign.yml

```diff
@@ -0,0 +1,311 @@
+        url: http://ftp.acc.umu.se/pub/GNOME/sources/pygobject/3.28/pygobject-3.28.2.tar.xz
+        sha256: ac443afd14fcb9ff5744b65d6e2b380e70510278404fb8684a9b9fb089e6f2ca
+
+  - name: swig
+    config-opts:
+      - "--without-boost"
+      - "--without-pcre"
+    sources:
+      - type: archive
+        url: http://prdownloads.sourceforge.net/swig/swig-3.0.12.tar.gz
+        sha256: 7cf9f447ae7ed1c51722efc45e7f14418d15d7a1e143ac9f09a668999f4fc94d
+
+  - name: dbus-python
+    ensure-writable:
+      - easy-install.pth
+      - setuptools.pth
+    sources:
+      - type: archive
+        url: https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.2.4.tar.gz
+        sha256: e2f1d6871f74fba23652e51d10873e54f71adab0525833c19bad9e99b1b2f9cc
+
+  - name: avahi
+    # hrm, I don't want to install this, but we need the python module
+    disabled: false
+    rm-configure: true
+    config-opts:
+      - "--enable-python"
+      - "--disable-static"
+      - "--disable-mono"
+      - "--disable-monodoc"
+      - "--disable-qt3"
+      - "--disable-qt4"
+      - "--disable-gtk"
+      - "--disable-gtk3"
+      - "--enable-core-docs"
+      - "--with-distro=none"
+      - "--with-systemdsystemunitdir=no"
+      - "--disable-libdaemon"
+      - "--disable-pygtk"
+      - "--disable-gdbm"
+    ensure-writable:
+      - easy-install.pth
+      - setuptools.pth
+    sources:
+      - type: archive
+        url: http://avahi.org/download/avahi-0.6.31.tar.gz
+        sha256: 8372719b24e2dd75de6f59bb1315e600db4fd092805bd1201ed0cb651a2dab48
+      - type: script
+        commands:
+        - autoreconf -fi
+        dest-filename: autogen.sh
+
+  - name: gstreamer
+    sources:
+      - type: archive
+        url: https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.12.3.tar.xz
+        sha256: d388f492440897f02b01eebb033ca2d41078a3d85c0eddc030cdea5a337a216e
+
+  - name: gst-plugins-base
+    sources:
+      - type: archive
+        url: https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.12.3.tar.xz
+        sha256: d3d37b8489d37fa0018973d850bd2067b98af335fef2fa543ee7d40359e3cea5
+
+  - name: gst-plugins-good
+    sources:
+      - type: archive
+        url: https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-1.12.3.tar.xz
+        sha256: 13e7f479296891fef5a686438f20ba7d534680becf2269ecc5ee24aa83b45f03
+
+  - name: gst-plugins-ugly
+    sources:
+      - type: archive
+        url: https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.12.3.tar.xz
+        sha256: e88ca584c94ea78eeecbf3af00ef7f134b66bdee7408aa4aa6c547235e060052
+
+  - name: gst-plugins-bad
+    build-options:
+      config-opts:
+        - "--enable-zbar"
+    sources:
+      - type: archive
+        url: https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.12.3.tar.xz
+        sha256: 36d059761852bed0f1a7fcd3ef64a8aeecab95d2bca53cd6aa0f08054b1cbfec
+    modules:
+      - name: zbar
```

Comment:
This needs to be reconfigured to compile on aarch64. Here's an example for 0.10 https://github.com/flathub/com.github.bilelmoussaoui.Authenticator/blob/efc4b2f25b3aa3d7f62496e9c2ece91fb0180d9e/com.github.bilelmoussaoui.Authenticator.json#L26 or I think it just works in 0.20 https://github.com/flathub/com.github.paolostivanin.OTPClient/blob/2ebb8605652c34bccd545df15bc1eb9ad9c708cc/com.github.paolostivanin.OTPClient.json#L57